### ISSUE TITLE
fix duration and date display in current running entries

### ISSUE THREAD

**Issue by beauraines** [beauraines/toggl-cli#62]
When running `toggl now` the duration and stop time are populated but not exactly right.

```
$ toggl now
#33 Update time-entries module for remaining v9 API changes #2911783204
Billable: false
Duration: 0h 0m
Project: Side Projects (#4599739)
Start: 2023-04-01 09:55
Stop: Invalid Date
```

I think that the duration is coming from what the API returns and its a fixed value, so if it comes back zero it should be calculated.

I think that the date conversion function is returning `Invalid Date` because its trying to convert null. 

I think that this should be handled in the display functions.

**Comment by beauraines** [comment]
Looking at the issue, this happens when a running time entry is edited. I'm not updating the `duration` when I edit the time entry

https://github.com/beauraines/toggl-cli/blob/0476db098c72f2997d916149dfdeb4f50d7a43ce/cmds/edit.mjs#L61-L68

```javascript
  params.created_with = appName
  params.at = dayjs().toISOString()
  project ? params.project_id = +project.id : undefined
  startTime ? params.start = startTime.toISOString() : undefined
  endTime ? params.stop = endTime.toISOString() : undefined
  argv.description ? params.description = argv.description : undefined
  debug(params)
  const timeEntry = await client.timeEntries.update(currentTimeEntry.id, params)
```

**Comment by beauraines** [comment]
Resolved in #90
